identity-account/src/account/builder.rs

```diff
@@ -140,36 +136,52 @@ impl AccountBuilder {
     Ok(Arc::clone(self.storage.as_ref().unwrap()))
   }
 
-  /// Apply configuration to the IOTA Tangle client for the given [`Network`].
+  /// Sets the IOTA Tangle [`Client`], this determines the [`Network`] used by the identity.
+  ///
```

Comment:
Consider adding a doc comment that all accounts created from the builder with this client set will share that client? Not as a caveat, but as a feature someone may want to use to save on memory?

Reply:
There's a doc comment regarding that on the `AccountBuilder` itself, I'll duplicate the relevant part for `client()` and `client_builder()`.